src/main/webapp/app/exam/manage/test-runs/test-run-management.component.ts

```diff
@@ -128,6 +128,19 @@ export class TestRunManagementComponent implements OnInit {
         return false;
     }
 
+    get examContainsExercises(): boolean {
```

Comment:
Could be shortened: `this.exam.exerciseGroups.some(exerciseGroup => exerciseGroup.exercises && exerciseGroup.exercises.length >  0)`

Reply:
Yeah, nice suggestion. I implemented it accordingly